alfasim_sdk/status.py

```diff
@@ -7,8 +7,13 @@
 @attr.s(kw_only=True)
 class ErrorMessage:
     """
-    The model_name and the message informed is displayed on the status monitor of ALFAsim.
-    The execution of a new simulation will not be allowed until the error is solved.
+    ErrorMessage allows the plugin to display a message over the status monitor, and signalize to the application to lock
```

Comment:
I think you meant _to *block* the simulation_

Reply:
😁